person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
- [ ] In your readme, there is a broken link. If I click the issues link in the readme it leads me to a broken page. Kindly fix the issue and make sure it's working as expected.